korman/nodes/node_python.py

```diff
@@ -85,6 +86,50 @@
     "ptAttribGrassShader": plFactory.ClassIndex("plGrassShaderMod"),
 }
 
+class StringVectorProperty(bpy.types.PropertyGroup):
+    value = StringProperty()
+
+class PlasmaAttributeArguments(bpy.types.PropertyGroup):
+    byObject = BoolProperty()
+    default = StringProperty()
+    options = CollectionProperty(type=StringVectorProperty)
+    range = FloatVectorProperty(size=2)
```

Comment:
Hides builtin function `range`, might not be a good idea

Reply:
I wondered about that.  I checked that it's not reserved, but I assumed this is why they used `rang` as the keyword.  I assumed it wouldn't be a problem since it's unlikely for the `range` function to apply on this class, but you're right that it's probably best to avoid that assumption and just call it `range_values` or something else irritatingly verbose.

Reply:
I personally think that the author was awaiting rangarok [sic]...